src/js/components/AppListLabelsFilterComponent.jsx

```diff
@@ -72,7 +72,11 @@ var AppListLabelsFilterComponent = React.createClass({
           }
         });
         return memo;
-      }, []);
+      }, [])
+      .sort((a, b) => {
+        let [labelKeyA, labelKeyB] = [Object.keys(a)[0], Object.keys(b)[0]];
```

Comment:
Yeah you're right. Also I'm always converting it back to an array when using these labels anyway ;)